src/lib/storage/index/b_tree/b_tree_index_impl.hpp

```diff
@@ -1,13 +1,6 @@
 #pragma once
 
-#ifdef __clang__
```

Comment:
How did you catch that?

Reply:
Before this PR, we explicitly ignored some clang compiler warnings with `#pragma clang diagnostic ignored`. Now, we add `cpp-btree` as system library to suppress these warnings (see `src/lib/CMakeLists.txt` line 730) as we also did it for other third-party libraries before.